bottles/backend/managers/manager.py

```diff
@@ -738,6 +739,28 @@ def process_bottle(bottle):
             if conf_file_yaml.get("run_in_terminal"):
                 conf_file_yaml["run_in_terminal"] = False
 
+            # Check if the path in the bottle config corresponds to the folder name
+            # if not, change the config to reflect the folder name
+            # if the folder name is "illegal" accross all platforms, rename the folder
+            sain_name = pathvalidate.sanitize_filepath(_name, platform='universal') # "universal" platform works for all filesystem/OSes
+            if conf_file_yaml["Custom_Path"] == False: # There shouldn't be problems with this
```

Comment:
Omitting `== False` should work.

```suggestion
            if conf_file_yaml["Custom_Path"]: # There shouldn't be problems with this
```

Reply:
I'm not checking if the key exists, I'm checking if the key is equal to the bool False